boards/nrf51dk/Makefile

```diff
@@ -1,26 +1,35 @@
 # Makefile for building the tock kernel for the nRF development kit
 
+SIZE?=arm-none-eabi-size
 OBJCOPY?=arm-none-eabi-objcopy
 OBJDUMP?=arm-none-eabi-objdump
 OBJDUMP_FLAGS+= --disassemble-all --source --disassembler-options=force-thumb -C --section-headers
 
 .PHONY: all
-all: target/target/release/nrf51dk target/target/release/nrf51dk.lst
+all: target/nrf51/release/nrf51dk
 
 .PHONY: doc
 doc:
-	@cargo doc --release --target=../../chips/nrf51/target.json
+	@cargo doc --release --target=nrf51.json
 
-.PHONY: target/target/release/nrf51dk
-target/target/release/nrf51dk:
-	@cargo build --release --target=../../chips/nrf51/target.json
+.PHONY: target/nrf51/release/nrf51dk
+target/nrf51/release/nrf51dk:
+	@cargo build --release --target=nrf51.json
+	@$(SIZE) $@
 
-target/target/release/nrf51dk.lst: target/target/release/nrf51dk
-	@$(OBJDUMP) $(OBJDUMP_FLAGS) $^ > $@
+.PHONY: target/nrf51/debug/nrf51dk
+target/nrf51/debug/nrf51dk:
+	@echo $(RUSTFLAGS)
+	@cargo build --target=nrf51.json
+	@$(OBJDUMP) $(OBJDUMP_FLAGS) $@ > target/nrf51/debug/nrf51dk.lst
```

Comment:
But, that's not right either. If you build the `dev` profile (what I'm doing with `make debug`) then the directory structure is `target/sam4l/debug/`. Adding it to the nrf51.json won't work with multiple profiles.